Neos.Neos/Documentation/References/ViewHelpers/TypoScript.rst

```diff
@@ -28,7 +28,7 @@ Arguments
 
 * ``typoScriptPackageKey`` (string, *optional*): The key of the package to load TypoScript from, if not from the current context.
 
-* ``typoScriptFilePathPattern`` (string, *optional*): Resource pattern to load TypoScript from. Defaults to: resource://@package/Private/TypoScript/
+* ``typoScriptFilePathPattern`` (string, *optional*): Resource pattern to load TypoScript from. Defaults to: resource://@package/Private/Fusion/
```

Comment:
what about this whole part `VieHelpers/TypoScript.rst`? There are still lots of occurrences of "TypoScript", so without having a closer look I assume that the documentation here isn't really consistent yet, right?